components/User/UserProfile/UserGroups.js

```diff
@@ -63,7 +63,18 @@ class UserGroups extends React.Component {
         const action = e.target.attributes.name.value;  //eg. changeGroup_2
         const groupid = action.split('_')[1];
 
-        this.context.executeAction(deleteUsergroup, {groupid: groupid});
+        swal({
+            titleText: 'Are you sure you want to delete this user group?',
+            type: 'warning',
+            showCancelButton: true,
+            confirmButtonColor: '#3085d6',
+            cancelButtonColor: '#d33',
+            confirmButtonText: 'Yes, delete it!'
+        }).then((accepted) => {
+            this.context.executeAction(deleteUsergroup, {groupid: groupid});
+            swal('User group successfully deleted');
```

Comment:
The user group hasn't been deleted yet. It has only been triggered to be deleted in the future (if successful).